google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPool.java

```diff
@@ -2761,4 +2790,77 @@ private void initMetricsCollection(MetricRegistry metricRegistry, List<LabelValu
         // TODO: Remove metric.
         ignored -> 0L);
   }
+
+  private void initOpenTelemetryMetricsCollection(
+      OpenTelemetry openTelemetry, Attributes attributes) {
+
+    if (openTelemetry != null) {
```

Comment:
I think it would be better to invert this if condition and do an early return. So like this:

```java
if (openTelemetry == null) {
  return;
}
```

That reduces the indentation of this method.